pandas/tests/indexes/test_interval.py

```diff
@@ -255,6 +255,11 @@ def test_insert(self):
         pytest.raises(ValueError, self.index.insert, 0,
                       Interval(2, 3, closed='left'))
 
+        # GH 18295
+        expected = self.index_with_nan
```

Comment:
can u add a fixture that hits multiple kinds of left (float,int,datetime); might be able to do this more generally in the interval tests

Reply:
done